### ISSUE TITLE
How does this relate to the main Automate documentation?

### ISSUE THREAD

**Issue by mrjcleaver** [chef-cft/wombat#229]
Re: https://docs.chef.io/chef_automate.html

What should we read first, under what circumstances should we use Wombat (only for customer demos?), at what point(s) of the Automate documentation does it take over, and of which parts of that documentation?

Can we use Wombat to set up a production install?

CC: @patcon

**Comment by andrewelizondo** [comment]
@mrjcleaver When @cheeseplus and I built out wombat it was purely to be used for building out isolated, repeatable demo clusters that would allow us to overlay specific content when demoing to customers. The thing that makes wombat somewhat simple comparatively to the standard way in which you build an automate cluster is that we predetermine all of the things that would've normally required us to code in coordination somewhere. 

It does however present a few problems.
1. you can't magically hook up a wombat built cluster to an existing chef server (you could but then you'd get yourself in a mess of spaghetti that would probably best be avoided entirely)
2. a wombat cluster makes a ton of assumptions (needfully) about the way it's deployed, it expects to be built in an isolated environment with predictable networking. Most people wanting to deploy a production automate cluster aren't okay with doing that.
3. static images are amazing for predictably standing up clusters that have keys and networking built into them and are easy to predict, but it also means you have include a bunch of wait time up front that might not be practical if you're building a single automate cluster

tl;dr - wombat is perfect for building demo scenarios, anything production should go by the docs

**Comment by mrjcleaver** [comment]
Thanks @andrewelizondo 

Is there a guided slideshare or video that shows what to demo thereafter?

Something on https://www.brighttalk.com/channel/11349/chef-software ?

Thanks,
  Martin.

**Comment by tpetchel** [comment]
Hi @mrjcleaver, there's a tutorial on Learn Chef that walks you through using it. We provide an AWS CloudFormation template that has wombat pre-built. We plan to update it to a newer version shortly.

https://learn.chef.io/automate/

**Comment by mrjcleaver** [comment]
Well, yes, I'd referred to the top of the /automate help in my opening post.

The answer I believe I needed was https://learn.chef.io/automate/visibility/bootstrap-nodes-in-your-demo-environment/ :) Thanks, anyway.

By way of documentation, I suppose this update has the pending change for things like replacing 'chordata' with animals.biz per the new build being inconsistent with the documentation.

![2016-10-08 17-48-03](https://cloud.githubusercontent.com/assets/59202/19216169/dd0b7d36-8d7f-11e6-8a70-f02d5bd6d7f1.png)

Re: in use for production purposes, actually I need to replace my existing Chef Server, but am mostly figuring out relative merits of Automate.

I believe its reasonable to expect a fully automated build of the Chef servers. It's kinda the mantra of Chef, after all. :) While I appreciate this is a work in progress, I do venture that "done" includes production, not just a demo. Whether production builds would use the same code base is another matter.

**Comment by cheeseplus** [comment]
Wombat is in no way officially sanctioned for anything but self-contained demos - full stop. Other stuff like Learn Chef is built on top of wombat but may have their own independent settings. There has been discussion about further internal use where we solve for the production flip but that remains out of scope at this time.

I'm going to close this one as answered.
